filters/named-entity-count/filter.py

```diff
@@ -0,0 +1,88 @@
+import operator
+from typing import List
+
+import spacy
+
+from initialize import spacy_nlp
+from interfaces.SentenceOperation import (
+    SentenceAndTargetOperation,
+    SentenceOperation,
+)
+from tasks.TaskTypes import TaskType
+
+"""
+A filter on text length (number of tokens).
+"""
+
+
+class NamedEntityCountFilter(SentenceOperation):
```

Comment:
It would be great if you can add DocString for classes and functions.

Reply:
thanks @aadesh11 , fixed this.